File: maxmin.cpp
#include<iostream>
using namespace std;
int main(){
    long long int a[5];
    long long int array[5];
    for(long long int i=1;i<=5;i++){
        cin>>a[i];
    }
    long long int sum=0,max=0,mix=0,count=1,k=0;
    for(long long int i=1;i<=5;i++){
        sum=0;
      for(long long int j=1;j<=5;j++){
        if(j!=count){
            sum+=a[j];
        }
      }
        array[k]=sum;
        k++;
        count++; 

    }

    for(long long int i=0;i<5;i++){
        for(long long int j=i+1;j<5;j++){
            if(array[i]>array[j]){
                long long int t;
                t=array[j];
                array[j]=array[i];
                array[i]=t;
            }
        }
    }
    cout<<array[0]<<" "<<array[4]<<"\n";
}

File: maxmin.py
a=input()
list=list(map(int,a.split()))
count=0
sun=0
array=[]
k=1
for i in range(5):
    sum=0
    for j in range(5):
        if(j!=count):
            sum+=list[j]

    array.append(sum)
    count=count+1

array=sorted(array)
print(array[0],'',array[4])
